src/main/java/xyz/nucleoid/plasmid/storage/FriendSystem/FriendListManager.java

```diff
@@ -0,0 +1,33 @@
+package xyz.nucleoid.plasmid.storage.FriendSystem;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/*
+    Unless you know what you are doing please do not touch this class TYSM || this is to (hopefully) prevent anything from breaking :P
+
+    if you have a legit reason for using this and want help ask I ran out of name ideas on the discord server
+
+ */
+public class FriendListManager {
```

Comment:
This should probably be implemented with vanilla's `PersistentState`. Can check `GameChannelManager` for an example of using that if needed